src/core_atmosphere/dynamics/mpas_atm_time_integration.F

```diff
@@ -27,9 +27,9 @@ module atm_time_integration
 
    use mpas_atm_iau  
 
-!#ifdef MPAS_OPENACC
-!   use openacc
-!#endif
+#ifdef MPAS_OPENACC
+   use openacc
+#endif
```

Comment:
@evanmacbride How is it that the code was working with this `use` statement commented-out? What error message were you getting with this commented-out? This would be useful information to include in the commit message for e4ea3627.